web/src/main/java/org/openshift/kieserver/web/redirect/PathPattern.java

```diff
@@ -113,9 +112,9 @@ public boolean matches(HttpServletRequest request) {
     public Map<String, String> getVariables(HttpServletRequest request) {
         Map<String, String> reqVars = new LinkedHashMap<String, String>();
         if (matches(request)) {
-            UriInfoImpl uriInfo = ServletUtil.extractUriInfo(request, request.getServletPath());
```

Comment:
Can you use the interface instead of the implementation, e.g. `javax.ws.rs.core.UriInfo`.

Also, if you could undo all your reformatting, it would make the diff's easier to parse.

Reply:
Also, I'm wondering if you just use the interface as @rcernich mentions above, then you might not have to change the dependencies in the pom (adding eap-runtime-artifacts).

Reply:
Thanks for the tip, I missed this class, seems to work.